.github/workflows/e2e-nvidia-l40s-x4.yml

```diff
@@ -138,10 +138,7 @@ jobs:
           python3.11 -m pip install packaging wheel setuptools-scm
           python3.11 -m pip install .[cuda] -r requirements-vllm-cuda.txt
 
-      - name: ilab system info
-        run: |
-          . venv/bin/activate
-          ilab system info
+          df -h
```

Comment:
Sadly this won't show up before it merges

Reply:
Yeah, good to have for the future in the meantime. I may add it to all runners actually.

Reply:
I think I actually could do it if I worked off a branch in this repo instead of my fork :thinking: